model/derived/gasoutputs.go

```diff
@@ -0,0 +1,70 @@
+package derived
+
+import (
+	"context"
+
+	"github.com/go-pg/pg/v10"
+	"go.opentelemetry.io/otel/api/global"
+	"go.opentelemetry.io/otel/api/trace"
+	"go.opentelemetry.io/otel/label"
+	"golang.org/x/xerrors"
+)
+
+type GasOutputs struct {
```

Comment:
This is what is persisted in the new `derived_gas_outputs` table

Reply:
Any reason for duplicating the message and receipt data in this table? Is it to avoid complicating our ingestion strategy?

Reply:
This is intended to be query optimized (indexed, normalized) format for supporting analysis and monitoring. This (query-optimized) schema may eventually live in another place so we shouldn't assume the presence of our ingestion schema. It also cleanly isolates our ingestion concerns from our analysis concerns.

Reply:
Columns are as requested in https://github.com/filecoin-project/sentinel-visor/issues/63